sway/input/cursor.c

```diff
@@ -63,6 +63,11 @@ struct sway_node *node_at_coords(
 		}
 	}
 
+	if (server.session_lock.lock) {
+		*surface = server.session_lock.lock->focused;
```

Comment:
`focused` represents the keyboard focus, but this codepath is about the pointer. Moreover, the surface-local coordinates `sx`/`sy` are left unset.

The session lock surfaces are added to `output->layers.session_lock`, so I wonder why this doesn't work?

cc @Nefsen402